test/contract/generatepin/testGeneratePin.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const expect = require('chai').expect;
+const config = require('app/config');
+const FormatUrl = require('app/utils/FormatUrl');
+const request = require('supertest');
+
+const VALIDATION_SERVICE_URL = config.services.validation.url;
+const VALID_SESSION_ID = '012233456789';
+const VALID_INTERNATIONAL_TEST_NUMBER = '+61437112945';
+const VALID_UK_WITH_PREFIX_TEST_NUMBER = '+447535538319';
+const VALID_UK_LOCAL_TEST_NUMBER = '07535538319';
+const VALID_PIN_CONTENT_LENGTH = '6';
+
+/* eslint no-console: 0 no-unused-vars: 0 */
```

Comment:
Can we remove `no-unused-vars` and also remove the unused `res` variable that it flags up

Reply:
Done, also updated addresslookup.js contract test as well.